diff --git a/node_modules/@redux-offline/redux-offline/lib/defaults/persist.native.js b/node_modules/@redux-offline/redux-offline/lib/defaults/persist.native.js
index db4e4ef..4f06a07 100644
--- a/node_modules/@redux-offline/redux-offline/lib/defaults/persist.native.js
+++ b/node_modules/@redux-offline/redux-offline/lib/defaults/persist.native.js
@@ -11,7 +11,7 @@ var _defineProperty2 = _interopRequireDefault(require("@babel/runtime/helpers/de
 
 var _reduxPersist = require("redux-persist");
 
-var _reactNative = require("react-native");
+var _asyncStorage = require("@react-native-async-storage/async-storage");
 
 function ownKeys(object, enumerableOnly) { var keys = Object.keys(object); if (Object.getOwnPropertySymbols) { var symbols = Object.getOwnPropertySymbols(object); if (enumerableOnly) symbols = symbols.filter(function (sym) { return Object.getOwnPropertyDescriptor(object, sym).enumerable; }); keys.push.apply(keys, symbols); } return keys; }
 
@@ -21,7 +21,7 @@ function _objectSpread(target) { for (var i = 1; i < arguments.length; i++) { va
 var _default = function _default(store, options, callback) {
   return (// $FlowFixMe
     (0, _reduxPersist.persistStore)(store, _objectSpread({
-      storage: _reactNative.AsyncStorage
+      storage: _asyncStorage
     }, options), callback)
   );
 };
diff --git a/node_modules/@redux-offline/redux-offline/src/defaults/detectNetwork.native.js b/node_modules/@redux-offline/redux-offline/src/defaults/detectNetwork.native.js
index 9bfccf7..9b07eb0 100644
--- a/node_modules/@redux-offline/redux-offline/src/defaults/detectNetwork.native.js
+++ b/node_modules/@redux-offline/redux-offline/src/defaults/detectNetwork.native.js
@@ -1,6 +1,6 @@
 /* eslint no-underscore-dangle: 0 */
-import { AppState } from 'react-native'; // eslint-disable-line
-import NetInfo from '@react-native-community/netinfo'; // eslint-disable-line
+import { AppState } from 'react-native';
+import NetInfo from '@react-native-community/netinfo';
 
 class DetectNetwork {
   constructor(callback) {
diff --git a/node_modules/@redux-offline/redux-offline/src/defaults/persist.native.js b/node_modules/@redux-offline/redux-offline/src/defaults/persist.native.js
index b8cc23a..38e73a4 100644
--- a/node_modules/@redux-offline/redux-offline/src/defaults/persist.native.js
+++ b/node_modules/@redux-offline/redux-offline/src/defaults/persist.native.js
@@ -1,8 +1,7 @@
 // @flow
 // $FlowIgnore
 import { persistStore } from 'redux-persist';
-// $FlowIgnore
-import { AsyncStorage } from 'react-native'; // eslint-disable-line
+import AsyncStorage from '@react-native-async-storage/async-storage';
 
 export default (store: any, options: {}, callback: any) =>
   // $FlowFixMe
